part_2_dynamic.md

```diff
@@ -4,66 +4,95 @@ title: Part 2
 permalink: /part_2/
 ---
 
-# Part 2 - Creating the Rails app
+# Part 2 - Introducing Rails
 
-* Open the terminal and create a new Rails app
+## Creating your (first) Rails app
+
+Open the terminal to create a new Rails app
 
 {% highlight bash %}
-rails new railsgirls-app
-cd railsgirls-app
+rails new railsgirls-blog
+cd railsgirls-blog
 {% endhighlight %}
 
-* Have a look how Rails generated a folder structure and a lot of files for you. There are two ways of doing that, in the terminal or with Finder/Explorer. Ask your coach about the details. Once you have seen them, move your ```about.html``` to the public folder.
+Find the project directory in the Finder/Explorer. Have a look how Rails generated a folder structure and a lot of files for you. Ask your coach about the details.
+
+
+## Starting the Rails server
+
+A Rails app can deliver dynamically generated content with the help of the Ruby programming language. This is why we can't just open a file anymore in the browser. We need a Rails server that will allow for all the magic to happen.
+
+You start your Rails server as follows. Open you terminal (in the `railsgirls-blog` folder) and type the following:
 
 {% highlight bash %}
-mv --your-downloads-folder-here--/about.html public/
-(i.e. mv ~/Downloads/about.html public/)
+rails server
 {% endhighlight %}
 
-## Start rails app
+Once it's started up, you can go visit your app already under
 
-* `rails server`
-* visit [http://localhost:3000/about.html](http://localhost:3000/about.html)
+[http://localhost:3000/](http://localhost:3000/).
 
-# Create pages controller
+It's not much but the page indicates that everything worked fine until now and the server is running.
 
-* open another terminal window and go to your app directory (`cd railsgirls-app`) to continue
-* `rails generate controller pages`
-* Open the file `config/routes.rb` in your text editor. The first line will look something like that:  `Rails.application.routes.draw do`
-* add the following as the second line of the file
+## The power of Rails: creating a scaffold
 
-{% highlight ruby %}
-get "/pages/about" => 'pages#about'
+One (not so secret) weapon of Rails is its ability to quickly get you started using scaffolds. Since we're trying to create blog here, let's put everything into place to create, update and show our blog articles. To do that type the following commands into your terminal:
```

Comment:
....it’s...
Add «To do that type the following commands into your terminal:» on a new line

Reply:
@borisrorsvort do you mean the `it's` in `of Rails is its ability`? I think it's fine according to https://data.grammarbook.com/blog/pronouns/1-grammar-error/.

The other one: is changed.